### PR TITLE
accuracy checker add missed license headers

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jeknins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: tools/accuracy_checker/accuracy_checker/launcher/tf_launcher.py [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
 import tensorflow as tf
 
 from .launcher import Launcher
 from ..config import BaseField, ListField, PathField, StringField, ConfigError, ConfigValidator
 
+
```
Comment (vladimir-dudnik) [inline tools/accuracy_checker/accuracy_checker/launcher/tf_launcher.py]: @eaidova why extra new line added?

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/launcher/tf_launcher.py]: It is standard code style to separate classes form import statements using 2 lines (instead 1 used in this file). It was applied by my IDE automatically.

Comment (vladimir-dudnik) [reply tools/accuracy_checker/accuracy_checker/launcher/tf_launcher.py]: ok, this is python coding style, accepted

**Pull request merged**